server/src/main/java/org/elasticsearch/client/Client.java

```diff
@@ -231,9 +231,9 @@
     BulkRequestBuilder prepareBulk();
 
     /**
-     * Executes a bulk of index / delete operations with default index and/or type
+     * Executes a bulk of index / delete operations with default index
      */
-    BulkRequestBuilder prepareBulk(@Nullable String globalIndex, @Nullable String globalType);
+    BulkRequestBuilder prepareBulk(@Nullable String globalIndex);
```

Comment:
NB I will open a separate PR for 7x to deprecate the method that takes a global type here.